apps/guardian-ui/src/components/LightningNode.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { CSSProperties, FC } from 'react';
+import { Box, Center, Flex, Stack, Text } from '@chakra-ui/react';
+import { LinkIcon } from '../assets/svgs/LinkIcon';
+import { LightningIcon } from '../assets/svgs/LightningIcon';
+
+export interface LightningNodeProps {
+  nodeName: string;
+  nodeId: string;
+  mempoolink: string;
+  capacityInSats: number;
+  capacityInDollars: number;
+  incomingFees: number;
+  outgoingFees: number;
+  successRate: number;
+}
+
+export const LightningNode: FC<LightningNodeProps> = ({
+  nodeName,
+  nodeId,
+  mempoolink,
+  capacityInSats,
+  capacityInDollars,
+  incomingFees,
+  outgoingFees,
+  successRate,
+}) => {
+  const titleStyles: CSSProperties = {
+    color: '#111827',
+    fontWeight: '600',
+  };
+
+  const subTextStyles: CSSProperties = {
+    color: '#6B7280',
+    fontWeight: '500',
+  };
+
+  return (
+    <Box
+      borderRadius='8px'
+      bgColor='#ffffff'
+      boxShadow='0px 1px 2px 0px rgba(0, 0, 0, 0.06), 0px 1px 3px 0px rgba(0, 0, 0, 0.10)'
+    >
+      <Flex p='16px 24px' gap='24px'>
+        <Center bgColor='#1570EF' h='48px' w='48px' borderRadius='6px'>
+          <LightningIcon color='#ffffff' />
```

Comment:
Prefer using theme variables instead of hex codes where possible. For instance, `#1570EF` is `theme.colors.blue[600]` and `#ffffff` is `theme.colors.white`.